src/components/ArchivePage/TagPageTemplate.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { Container, Typography } from '@material-ui/core';
+import EventInfoItem from './EventInfoItem';
+import { makeStyles } from '@material-ui/core/styles';
+import PageTitle from '../PageTitle/PageTitle';
+import HeadFooter from '../HeadFooter/HeadFooter';
+import SEO from '../SEO';
+import PropTypes from 'prop-types';
+
+const useStyles = makeStyles(theme => ({
+	container: {
+		marginTop: theme.spacing(4),
+		marginBottom: theme.spacing(8)
+	},
+	quarterItem: {
+		display: 'flex',
+		flexDirection: 'column',
+		width: '100%'
+	},
+	quarterEvent: {
+		padding: theme.spacing(2, 0)
+	},
+	subtitle: {
+		margin: theme.spacing(6, 0, 2)
+	},
+	tagColor: {
+		color: '#FF477E'
+	}
+}));
+
+function sortByQuarter(firstQuarter, secondQuarter) {
+	const [firstEventQuarter, firstEventYear] = firstQuarter.split(' ');
+	const [secondEventQuarter, secondEventYear] = secondQuarter.split(' ');
+
+	const quarterOrder = ['Winter', 'Spring', 'Summer', 'Fall'];
+
+	if (firstEventYear === secondEventYear) {
+		return quarterOrder.indexOf(secondEventQuarter) - quarterOrder.indexOf(firstEventQuarter);
+	}
+	return parseInt(secondEventYear) - parseInt(firstEventYear);
+}
+
+function getQuarterList(quarterEvents) {
+	const quarterList = [];
+	for (const property in quarterEvents) {
```

Comment:
It's generally preferred to do `for (const property of Object.keys(quarterEvents))` rather than `for`-`in` loop. See https://stackoverflow.com/questions/43446241/for-in-or-for-of-object-keys.